__tests__/helpers/supabase-mock.ts

```
@@ -0,0 +1,123 @@
+    'maybeSingle',
+    'filter',
+    'match',
+    'range',
+  ];
+
+  for (const method of methods) {
+    chain[method] = vi.fn().mockReturnValue(chain);
+  }
+
+  // Thenable so `await` resolves to the configured response
+  chain.then = vi.fn((resolve: (value: SupabaseResponse) => void) =>
```

Biome (2.3.14)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)